E
======================================================================
ERROR: palindrome_integer_testSuite (unittest.loader._FailedTest.palindrome_integer_testSuite)
----------------------------------------------------------------------
ImportError: Failed to import test module: palindrome_integer_testSuite
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/unittest/loader.py", line 419, in _find_test_path
    module = self._get_module_from_name(name)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/unittest/loader.py", line 362, in _get_module_from_name
    __import__(name)
  File "/app/palindrome_integer_testSuite.py", line 4, in <module>
    print(palindrome_integer_1(x))
          ^^^^^^^^^^^^^^^^^^^^
NameError: name 'palindrome_integer_1' is not defined


----------------------------------------------------------------------
Ran 1 test in 0.005s

FAILED (errors=1)
Command exited with non-zero status 1
	Command being timed: "python3 -m unittest discover -s . -p *test*.py"
	User time (seconds): 0.54
	System time (seconds): 0.45
	Percent of CPU this job got: 12%
	Elapsed (wall clock) time (h:mm:ss or m:ss): 0:08.34
	Average shared text size (kbytes): 0
	Average unshared data size (kbytes): 0
	Average stack size (kbytes): 0
	Average total size (kbytes): 0
	Maximum resident set size (kbytes): 14720
	Average resident set size (kbytes): 0
	Major (requiring I/O) page faults: 0
	Minor (reclaiming a frame) page faults: 2502
	Voluntary context switches: 128
	Involuntary context switches: 291
	Swaps: 0
	File system inputs: 0
	File system outputs: 576
	Socket messages sent: 0
	Socket messages received: 0
	Signals delivered: 0
	Page size (bytes): 4096
	Exit status: 1
